PROTO_tests/tests/extrema_test.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+
+
```

Comment:
isort wants to remove one of these blank lines

Reply:
reran isort